core/services/feeds/service.go

```diff
@@ -107,6 +107,9 @@ type Service interface {
 	ListSpecsByJobProposalIDs(ctx context.Context, ids []int64) ([]JobProposalSpec, error)
 	RejectSpec(ctx context.Context, id int64) error
 	UpdateSpecDefinition(ctx context.Context, id int64, spec string) error
+
+	// UnsafeSetConnectionsManager Only for testing
+	UnsafeSetConnectionsManager(ConnectionsManager)
```

Comment:
@asoliman92 Have you checked if it's still needed? Locally `integration_tests.go` and `integration_legacy_test.go` works for me when removed

Checking here https://github.com/smartcontractkit/ccip/pull/1253

Reply:
Ignore that, it's needed for CLO tests I created long time ago :P (Test_CLOSpecApprovalFlow)